package-lock.json

```diff
@@ -10536,6 +10536,7 @@
         "moment": "^2.13.0",
         "node-uuid": "^1.4.7",
         "normalize.css": "^4.1.1",
+        "octopus-library": "file:",
```

Comment:
@twerthi - is this expected?

Reply:
That's a good question, I'm not sure.  I hadn't noticed it was included, I didn't run anything that would have produced it, just updated the JSON file.

Reply:
I'm mindful of changes to the package-lock file generally. If you arent expecting it to change, I'd probably say revert it?

Reply:
Pulled file from main branch and overwrote PR branch.